### PR TITLE
Added py scripts to parse resource data

### PR DESCRIPTION
**Pull request by edmkitty**
Co-authored-by: Ethan Strominger <ethanstrominger2@gmail.com>

## Release Checklist

- [ ] This code has unit tests
- [ ] I have a plan to verify that these changes are working as expected after deploy
- [ ] I have a plan for how to revert, rollback, or disable these changes if things are not working as expected

## Security Checklist

This PR creates, modifies, or deletes:

- [ ] API routes: API routes, parameters, or user authorization
- [ ] Authentication: Authentication mechanism
- [ ] Credentials: Server side credentials, or secrets in configuration / source code
- [ ] Cryptography: Encryption, hashing, certificates, signatures, random numbers, etc.
- [ ] User data: personal information handling, logs, error messages, etc.

### CODE REVIEW

File: map/README.md [hunk 1]
```diff
@@ -14,6 +14,16 @@ You may find instructions on how to contribute data to the map [here](src/data/R
...
 
+### Build @reach4help/model dependency:
+
+`cd ../model` `yarn build`
+
+Without this step, you will see this error:
+
+```
+Module not found: Can’t resolve ‘@reach4help/model/lib/markers’ in ‘/Users/ethanstrominger/projects/reach4help/map/src/data’
+```
```
Comment (mcverter) [inline map/README.md]: Where is this pathname coming from?  It looks like it is a problem in @ethanstrominger 's local environment.  
```py
python delhi_parse.py
``` 

works fine on my machine

Comment (edmkitty) [reply map/README.md]: Oops, I did not update that to ~. I had this error on my local machine as well.

Comment (mcverter) [reply map/README.md]: What did you do that generated this error? Didn't you just execute the script?

**Review by mcverter — APPROVED** [review]
Great job!

**Pull request merged**